models/user.go

```diff
@@ -76,6 +77,20 @@ var (
 	ErrUnsupportedLoginType = errors.New("Login source is unknown")
 )
 
+// VisibleType defines the visibility (Organization only)
+type VisibleType int
+
+const (
+	// VisibleTypePublic Visible for everyone
+	VisibleTypePublic VisibleType = iota + 1
```

Comment:
Are we starting at 1 here to allow old organisations to be undefined?

Reply:
Updated to start at 0.